dev/plot_test.py

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I made this simple test file to test this branch instead of running and editing the tutorial